apollo-runtime/src/main/java/com/apollostack/android/GraphqlService.java

```diff
@@ -0,0 +1,14 @@
+package com.apollostack.android;
+
+import android.database.Observable;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/** A {@link retrofit2.Retrofit} service for executing GraphQL requests. */
+public interface GraphqlService {
+  // TODO: Replace the wildcard with an interface once apollo-api module is merged.
```

Comment:
Right, I'll add a common interface to all the `Data` objects, so we can use that here